engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/activation-key-details-info.controller.js

```diff
@@ -61,6 +67,12 @@ angular.module('Bastion.activation-keys').controller('ActivationKeyDetailsInfoCo
             $scope.disableEnvironmentSelection = false;
         };
 
+        $scope.clearContentView = function (activationKey) {
+            activationKey["content_view_id"] = null;
+            activationKey["content_view"] = null;
+            $scope.saveContentView(activationKey);
+        };
```

Comment:
shoudln't this unset the lifecycle environment too?

Reply:
@jlsherrill good point. Will look into it. One of the issues is that with the current environment selector resetting is harder than setting `activationKey.environment = null`. 
@jlsherrill @rohoover should I just get rid of it ?